src/Main.java

```diff
@@ -61,13 +91,101 @@ public static Employee findEmployeeWithMaxSalary() {
         return employeeWithMaxSalary;
     }
 
-    public static double calculateAverageSalary() {
+    private static double calculateAverageSalary() {
         return Math.round((calculateTotalSumSalary() / employees.length) * 100.0) / 100.0;
     }
 
-    public static void printFullNameEmployee() {
+    private static void printFullNameEmployee() {
         for (Employee employee : employees) {
             System.out.println(employee.getFullName());
         }
     }
-}
\ No newline at end of file
+
+    private static void indexSalaryEmployee(double percent) {
+        double indexSalary;
+        for (Employee employee : employees) {
+            indexSalary = employee.getSalary() + employee.getSalary() * percent / 100;
+            employee.setSalary(Math.round(indexSalary * 100.0) / 100.0);
+        }
+    }
+
+    private static Employee findEmployeeWithMinSalaryInDepartment(int departmentId) {
+        Employee employeeWithMinSalary = null;
+        for (Employee employee : employees) {
+            if (departmentId == employee.getDepartment()) {
+                if (employeeWithMinSalary == null || employee.getSalary() < employeeWithMinSalary.getSalary()) {
```

Comment:
вложенные if можно и нужно объединить в один
```suggestion
            if (departmentId == employee.getDepartment() && (employeeWithMinSalary == null || employee.getSalary() < employeeWithMinSalary.getSalary())) {
```